strr-web/composables/useChipFlavour.ts

```diff
@@ -10,11 +10,11 @@ export const useChipFlavour = () => {
     text: tStatuses(translationKey)
   })
 
-  const examinerOrHostStatusMap = (flavour: AlertsFlavourE, key: string) => ({
+  const examinerOrHostStatusMap = (flavour: AlertsFlavourE, key: string, hostSpecificKey?: string) => ({
     alert: flavour,
     text: isExaminer
       ? tStatuses(`examinerStatuses.${key}`)
-      : tStatuses(`hostStatuses.${key}`)
+      : tStatuses(`hostStatuses.${key || hostSpecificKey}`)
```

Comment:
It probably should be the other way around. If `hostSpecificKey` not there, use the `key`.

Reply:
@shaangill025 why I say that is that because `key` will always be there as a first param, so `hostSpecificKey` will not take over

Reply:
Renamed `key` to `commonKey`.  I think that other way around is not the correct approach.